build.gradle

```diff
@@ -862,6 +922,7 @@ task buildAll {
   dependsOn getTasksByName('product', true).collect { it.path }
   dependsOn getTasksByName('testClasses', true).collect { it.path }
   dependsOn distTar, distZip
+  dependsOn distInstallers
```

Comment:
I think distInstallers dep can be removed from buildAll because that is commonly used by devs to build all the test jars (hydra etc) and this will increase the build time (and requires rpm/deb packages to be installed?). Even the distTar, distZip tasks can be removed and replaced by dependency on only "product".